app/bundles/LeadBundle/Tests/Entity/LeadFieldRepositoryTest.php

```diff
@@ -30,14 +31,16 @@ protected function setUp(): void
 
     public function testCompareDateValueForContactField(): void
     {
-        $contactId        = 12;
-        $fieldAlias       = 'date_field';
-        $value            = '2019-04-30';
-        $builderAlias     = $this->createMock(QueryBuilder::class);
-        $builderCompare   = $this->createMock(QueryBuilder::class);
-        $statementAlias   = $this->createMock(Statement::class);
-        $statementCompare = $this->createMock(Statement::class);
-        $exprCompare      = $this->createMock(ExpressionBuilder::class);
+        $contactId                = 12;
+        $fieldAlias               = 'date_field';
+        $value                    = '2019-04-30';
+        $builderAlias             = $this->createMock(QueryBuilder::class);
+        $builderCompare           = $this->createMock(QueryBuilder::class);
+        $statementAlias           = $this->createMock(Statement::class);
```

Comment:
Mock of Result class is added then Mock of Statement class should be removed.
$statementAliasResul and $statementCompareResult should be used instead $statementAlias and $statementCompare.
Same need to be done in other places also in this file.
Please take care of extra spaces every where.

Reply:
Fixed in commit 3eb163cc33db937f70e871a4d07ebc143fd10bb3